### PR TITLE
bug/DEVENGAGE-1420 varType bug in CLI

### PR DESCRIPTION
**Pull request by dginty4**

### CODE REVIEW

File: resources/sdk/clisdkclient/templates/api.mustache [hunk 1]
```diff
@@ -73,6 +73,11 @@ var {{operationId}}Cmd = &cobra.Command{
 		if len(queryParams) > 0 {
 			urlString = fmt.Sprintf("%v?", path)
 			for k, v := range queryParams {
+                if k == "varType" {
+                    k = "type"
+                    urlString += fmt.Sprintf("%v=%v&", url.QueryEscape(strings.TrimSpace(k)), url.QueryEscape(strings.TrimSpace(v)))
+                    continue
+                }
```
Comment (charliecon) [inline resources/sdk/clisdkclient/templates/api.mustache:80]: Looks like the indentation is a bit off. You can fix this by opening this file in vs code and tabbing this block in and out again

**Review by carnellj-genesys — APPROVED** [review]

File: resources/sdk/clisdkclient/templates/api.mustache [hunk 2]
```diff
@@ -73,6 +73,11 @@ var {{operationId}}Cmd = &cobra.Command{
 		if len(queryParams) > 0 {
 			urlString = fmt.Sprintf("%v?", path)
 			for k, v := range queryParams {
+				if k == "varType" {
+					k = "type"
+					urlString += fmt.Sprintf("%v=%v&", url.QueryEscape(strings.TrimSpace(k)), url.QueryEscape(strings.TrimSpace(v)))
+					continue
```
Comment (charliecon) [inline resources/sdk/clisdkclient/templates/api.mustache:79]: I don't think there's any need for lines 78 and 79, unless I'm missing something

**Review by michaeljroddy — COMMENTED** [review]
another way to do this might be to just check if the `urlString` contains `varType` (after the for loop) and if it does, replace it with `type` like this:

```
if strings.Contains(urlString, "varType") {
        urlString = strings.Replace(urlString, "varType", "type", -1)
}
```

@charliecon @dginty4 thoughts?

**Comment by charliecon** [comment]
> another way to do this might be to just check if the `urlString` contains `varType` (after the for loop) and if it does, replace it with `type` like this:
> 
> ```
> if strings.Contains(urlString, "varType") {
>         urlString = strings.Replace(urlString, "varType", "type", -1)
> }
> ```
> 
> @charliecon @dginty4 thoughts?

Yeah we're probably splitting hairs but I suppose less nesting is always better

**Comment by dginty4** [comment]
> > another way to do this might be to just check if the `urlString` contains `varType` (after the for loop) and if it does, replace it with `type` like this:
> > ```
> > if strings.Contains(urlString, "varType") {
> >         urlString = strings.Replace(urlString, "varType", "type", -1)
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > @charliecon @dginty4 thoughts?
> 
> Yeah we're probably splitting hairs but I suppose less nesting is always better

I might just change it to this, I think its simpler

**Review by charliecon — APPROVED** [review]

**Review by michaeljroddy — APPROVED** [review]

**Pull request merged**